assets/queries/cloudFormation/docdb_cluster_master_user_password_rule/test/negative.yaml

```diff
@@ -0,0 +1,125 @@
+Parameters:
```

Comment:
There should be always a space between a property and it's value

This document is not a valid sample. Check the bottom of the page of the [AWS Cloudformation template reference documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-docdb-dbcluster.html
) for DocDB DBCluster samples.

Use [linters](https://github.com/martysweet/cfn-lint) and/or [YAML validators](https://jsonformatter.org/yaml-validator/9257f9) to validate the samples